stripe/src/main/java/com/stripe/android/view/PaymentAuthWebViewActivityViewModel.kt

```diff
@@ -1,16 +1,26 @@
 package com.stripe.android.view
 
+import android.app.Application
 import android.content.Intent
 import android.net.Uri
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
+import com.stripe.android.AnalyticsEvent
+import com.stripe.android.Logger
+import com.stripe.android.PaymentConfiguration
 import com.stripe.android.StripeIntentResult
 import com.stripe.android.auth.PaymentAuthWebViewContract
+import com.stripe.android.networking.AnalyticsDataFactory
+import com.stripe.android.networking.AnalyticsRequest
+import com.stripe.android.networking.AnalyticsRequestExecutor
 import com.stripe.android.payments.PaymentFlowResult
 import com.stripe.android.stripe3ds2.init.ui.StripeToolbarCustomization
 
 internal class PaymentAuthWebViewActivityViewModel(
-    private val args: PaymentAuthWebViewContract.Args
+    private val args: PaymentAuthWebViewContract.Args,
```

Comment:
not urgent, but have we considered daggerize the SDK? It could probably save us quite a bit of Factory definition and boilerplate calls.

Reply:
Discussed offline